test/sql/types/list/list_concat_null.test

```diff
@@ -28,3 +34,18 @@ query I
 select concat([42]::INT[1], [43]::INT[1], NULL::INT[1], [44]::INT[1], NULL::INT[1], [45]::INT[1]);
 ----
 [42, 43, 44, 45]
+
+query I
+select list_concat([42]);
+----
+[42]
+
+query I
+select list_concat([42], [43], [], [44], [], [45]);
+----
+[42, 43, 44, 45]
+
+query I
+select list_concat([42]::INT[1], [43]::INT[1], NULL::INT[1], [44]::INT[1], NULL::INT[1], [45]::INT[1]);
```

Comment:
Maybe we can also try to concat more than two arrays with different sizes (what is the expected behaviour?). And maybe arrays in lists with different sizes, which should fail.